src/App.js

```diff
@@ -10,10 +11,35 @@ const tabs = [
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedTab: tabs[0],
+  };
+
+  selectTab = tab => this.setState({ selectedTab: tab });
+
+  render() {
+    return (
+      <div className="App">
+        <h1>
+          Selected tab is&nbsp;
+          {tabs.find(
+            tab => this.state.selectedTab.id === tab.id,
+          ).title
+          }
```

Comment:
```suggestion
    const { selectedTab } = this.state;
    
    return (
      <div className="App">
        <h1>
          Selected tab is&nbsp;
          {selectedTab.title}
```
никто и никогда не заметит что после `find` ты ещё и получаешь свойство `title`